platforms/atmelmegaavr_extra.rst

```diff
@@ -25,6 +25,48 @@ incorrect upload flags. It's highly recommended to use the
 :ref:`projectconf_upload_command` option that gives the full control over flags used
 for uploading. Please read :ref:`atmelavr_upload_via_programmer` for more information.
 
+Upload using pymcuprog serialUPDI
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+``pymcuprog`` is a Python-based CLI tool for programming AVR devices with UPDI interface
+using either a Microchip debugger (Atmel-ICE, PICkit4, Snap, Curiosity Nano) or via a
```

Comment:
"Curiosity Nano" is the public name? Or PKOB Nano?

Reply:
Good point.  Maybe I should put all three to be clear: Curiosity Nano debugger / PKOB nano / nEDBG